Classes/DRYBaseNavigationManager.h

```diff
@@ -3,22 +3,34 @@
 // Copyright (c) 2015 AppFoundry. All rights reserved.
 //
 
-@import Foundation;
-@import UIKit;
+#import <UIKit/UIKit.h>
 
 #import "DRYNavigator.h"
 
+@protocol DRYNavigatorFactory;
+@protocol DRYViewControllerInitializerFactory;
+@protocol DRYFlowTranslationDataSource;
 @protocol DRYNavigationTranslationDataSource;
 @class DRYNavigationDescriptor;
 
-@protocol DRYNavigatorFactory;
+@protocol DRYNavigationManager <NSObject>
 
-@interface DRYBaseNavigationManager : NSObject
+- (UIViewController *)initialViewControllerForFlowWithIdentifier:(NSString *)identifier parameters:(NSDictionary *)parameters error:(NSError **)error;
+- (void)navigateWithNavigationIdentifier:(NSString *)identifier parameters:(NSDictionary *)parameters hostViewController:(UIViewController *)hostViewController errorHandler:(DRYNavigationErrorHandler)errorHandler successHandler:(DRYNavigationSuccessHandler)successHandler;
 
-- (instancetype)initWithNavigationTranslationDataSource:(id <DRYNavigationTranslationDataSource>)navigationTranslationDataSource;
```

Comment:
We'll move to 3.0 for this one. If we don't we have API inconstancy: one datasource is required, the other would not be...